### PR TITLE
data: debadityasingh8972

### PR DESCRIPTION
**Pull request by debadityasingh8972**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/8504"></a>

### CODE REVIEW

File: data/debadityasingh8972.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Debaditya Singh",
+  "displayStatsPublic": ture,
+  "type": "personal",
+  "bio": "Software Developer Fresher Looking for opportunities to test my skills | Full-stack Web Developer | Open-Source Contributor | Blockchain | Ethereum | DeFi | 3⭐ @LeetCode |"
+  "avatar": "https://github.com/debadityasingh8972.png",
```
Comment (kumarsonsoff3) [inline data/debadityasingh8972.json:6]: We no longer need the `avatar`. `type`, & `displayStatsPublic` keys. Please remove these from your `json`! 

```suggestion
  "bio": "Software Developer Fresher Looking for opportunities to test my skills | Full-stack Web Developer | Open-Source Contributor | Blockchain | Ethereum | DeFi | 3⭐ @LeetCode |",
```

File: data/debadityasingh8972.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/debaditya5972?t=JmTh6wWhFzsbRK9q6M-7Lg&s=08",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Connect with me on Github",
+      "url": "https://github.com/debadityasingh8972",
+      "icon": "FaGithub"
+    },
```
Comment (kumarsonsoff3) [inline data/debadityasingh8972.json:17]: Also, remove this trailing comma `,` here.
```suggestion
    }
```

**Review by kumarsonsoff3 — DISMISSED** [review]

**Review by debadityasingh8972 — COMMENTED** [review]
Changes added

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**